install.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -e
```

Comment:
early exit for the install script. without this, the script basically never failed in the background jobs (the ones startet with an &). This will lead to the early exit from the background jobs.